docs/content/SegmentedControl.mdx

```diff
@@ -43,7 +43,7 @@ description: Use a segmented control to let users select an option from a short
 ### With labels hidden on smaller viewports
 
 ```jsx live drafts
-<SegmentedControl aria-label="File view" variant={{narrow: 'hideLabels', regular: 'none'}}>
+<SegmentedControl aria-label="File view" variant={{narrow: 'hideLabels', regular: 'default'}}>
```

Comment:
non-blocking: `hideLabels` looks like a boolean type as opposed to `dropdown` variant. Wouldn't it be more consistent to use `icons` instead?